Classes/Controller/ToolboxController.php

```diff
@@ -268,6 +268,38 @@ public function renderScoreTool()
         }
     }
 
+    /**
+     * List of common web image mimetypes
+     * The MIMETYPE attribute must specify the media type of the digital representation. All web-compatible formats as per RFC2046 are allowed.
+     */
+    private const IMAGE_MIMETYPES = [
+        "image/jpeg",
+        "image/jpg",
+        "image/png",
+        "image/gif",
+        "image/bmp",
+        "image/tiff",
+        "image/x-tiff",
+        "image/webp",
+        "image/svg+xml",
+        "image/vnd.microsoft.icon",
+        "image/x-icon",
+        "image/heif",
+        "image/heic",
+        "image/vnd.adobe.photoshop",
+        "image/x-xbitmap",
+        "image/x-xpixmap",
+        "image/jp2",
+        "image/jpx",
+        "image/jpm",
+        "image/mj2",
+        "image/x-portable-anymap",
+        "image/x-portable-bitmap",
+        "image/x-portable-graymap",
+        "image/x-portable-pixmap",
+        "application/pdf"
```

Comment:
Isn't this mimetype a bit misleading, cause it`s no image mime type? Or should the name of the list therefore be adapted?

Reply:
If you mean "application/pdf". Actually yes, it is. See my previous comment. It should accept most of the image mime-types. But if some of them are misleading, they should be removed. I can update the list.